django_sorcery/db/fields.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, unicode_literals
+
+import sqlalchemy as sa
+
+from django.core import validators
+
+
+class Field(sa.Column):
+    default_validators = []
+    type_class = None
+
+    def __init__(self, *args, **kwargs):
+        column_kwargs = self.get_column_kwargs(**kwargs)
+        type_kwargs = self.get_type_kwargs(**kwargs)
+        type_class = self.get_type_class(**kwargs)
+
+        kwargs["type_"] = type_class(**type_kwargs)
+        super(Field, self).__init__(*args, **column_kwargs)
+
+    def get_type_kwargs(self, **kwargs):
+        return {}
+
+    def get_column_kwargs(self, **kwargs):
+        info = kwargs.setdefault("info", {})
+        info["validators"] = self.default_validators + kwargs.pop("validators", [])
+        return kwargs
+
+    def get_type_class(self, **kwargs):
+        return self.type_class
+
+
+class AutoField(Field):
+    type_class = sa.Integer
```

Comment:
django does it the other way around. a column on a model is tied to a DB type and each field in `_meta` can translate itself to a form field. dont think its a good idea to tie column types to form fields....